src/pages/components/OperatorSelector.tsx

```diff
@@ -18,7 +18,7 @@ const OperatorSelector = ({
   const [operators, setOperators] = useState<Operator[]>([])
   useEffect(() => {
     const majorOperatorsIds = ['3', '5', '15', '18', '25']
-    RELEVANT_OPERATORS.then((resultObj) =>
+    RELEVANT_OPERATORS().then((resultObj) =>
```

Comment:
now, when it's a function and not a constant, using camalCase would be better
```suggestion
    getRelevantOperators().then((resultObj) =>
```